### PR TITLE
Add secrets to languages

### PR DESCRIPTION
**Pull request by bigfluffycookie**
fixes https://github.com/SonarSource/sonarlint-visualstudio/issues/3663

### CODE REVIEW

File: src/Core.UnitTests/LanguageTests.cs [hunk 1]
```diff
@@ -67,6 +67,9 @@ public void Language_IsSupported_SupportedLanguage_IsTrue()
             Language.VBNET.IsSupported.Should().BeTrue();
             Language.Cpp.IsSupported.Should().BeTrue();
             Language.C.IsSupported.Should().BeTrue();
```
Comment (bigfluffycookie) [inline src/Core.UnitTests/LanguageTests.cs:69]: I felt weird only adding the secret tests so I added the js and ts stuff aswell.

**Comment by bigfluffycookie** [comment]
Unsure if I updated the submodule correctly but I think i did not

File: src/Core.UnitTests/LanguageTests.cs (hunk 1 above)
Comment (duncanp-sonar) [reply src/Core.UnitTests/LanguageTests.cs:69]: Thanks! If you spot a test hole when changing a component, it's a practice to add some tests if it doesn't take too much effort.

File: src/Integration/Binding/BindingController.cs [hunk 2]
```diff
@@ -178,7 +178,8 @@ void IBindingWorkflowExecutor.BindProject(BindCommandArgs bindingArgs)
                     Language.C,
                     Language.Cpp,
                     Language.Js,
-                    Language.Ts
+                    Language.Ts,
+                    Language.Secrets
```
Comment (duncanp-sonar) [inline src/Integration/Binding/BindingController.cs:182]: Did you mean to include this change in the PR?

**Comment by duncanp-sonar** [comment]
> Unsure if I updated the submodule correctly but I think i did not

I think it's ok. We can discuss it tomorrow.

In hindsight, we should have created a feature branch in the `sonarqube-webclient-dotnet` repo and merged the language change to that first (in the same way Rita and Georgii have a `branch-sse` feature branch  in that repo for their changes).
However, our change is minor and non-impacting, so it won't have done any harm.

File: src/Integration/Binding/BindingController.cs (hunk 2 above)
Comment (bigfluffycookie) [reply src/Integration/Binding/BindingController.cs:182]: Hm actually, I am not sure as I dont fully understand what the binding config provider is. Do you think it makes more sense to include it in the next pr?

**Review by duncanp-sonar — APPROVED** [review]

**Pull request merged**